connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/adhoctoolsschema/resolver/schema/AdHocToolSchemaGenerator.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.agenticai.adhoctoolsschema.resolver.schema;
+
+import io.camunda.connector.agenticai.adhoctoolsschema.feel.FeelInputParam;
+import java.util.List;
+import java.util.Map;
+
+public interface AdHocToolSchemaGenerator {
```

Comment:
FYI this is singular Tool Schema opposed to plural Tools Schema like the function as it is only handling the schema for a single tool.